server/elm/src/Pages/Reports/View.elm

```diff
@@ -0,0 +1,235 @@
+module Pages.Reports.View exposing (view)
+
+import App.Types exposing (Language, Site)
+import AssocList as Dict exposing (Dict)
+import Backend.Model exposing (ModelBackend)
+import Backend.Reports.Model exposing (Gender(..), ReportsData)
+import Date exposing (Interval(..), Unit(..))
+import DateSelector.SelectorPopup exposing (viewCalendarPopup)
+import Gizra.Html exposing (emptyNode)
+import Gizra.NominalDate exposing (NominalDate, formatDDMMYYYY)
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick)
+import Maybe.Extra exposing (isJust)
+import Pages.Reports.Model exposing (..)
+import Pages.Reports.Utils exposing (..)
+import Pages.Utils exposing (viewSelectListInput, wrapSelectListInput)
+import Translate exposing (TranslationId, translate)
+import Utils.Html exposing (viewModal)
+
+
+view : Language -> NominalDate -> ModelBackend -> Model -> Html Msg
+view language currentDate modelBackend model =
+    case modelBackend.reportsData of
+        Just (Ok data) ->
+            viewReportsData language currentDate data model
+
+        Just (Err err) ->
+            text <| Debug.toString err
```

Comment:
It might be better to handle errors more gracefully or provide a user-friendly message.

Reply:
Oh, if we get here, APP is broken.
As we develop the tool for admins, they'll know how to find us :)